modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -6,51 +6,118 @@
 [id="talo-platform-prepare-end-to-end_{context}"]
 = End-to-end procedures for updating clusters in a disconnected environment
 
+If you have deployed spoke clusters with distributed unit (DU) profiles using the GitOps ZTP with Topology Aware Lifecycle Operator pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure describes how to upgrade your spoke clusters and Operators.
+
 [id="talo-platform-prepare-for-update_{context}"]
 == Preparing for the updates
 
-If both the hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
+If both the hub and the spoke clusters are running {product-title} 4.9, you must update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
 
 [id="talo-platform-prepare-for-update-env-setup_{context}"]
 == Setting up the environment
 
-If you deployed ZTP in a disconnected environment as instructed in the "Preparing the disconnected environment" section in "Deploying distributed units at scale in a disconnected environment", you need to do the following steps before TALO can successfully perform update operations:
+The TALO can perform both platform and Operator updates.
 
-TALO can perform both platform and Operator updates:
+You must mirror both the platform image and Operator images that you want to update to in your mirror registry before you can use the TALO to update your disconnected clusters. Complete the following steps to mirror the images:
 
-* For platform updates, you have to do the following:
+* For platform updates, you must perform the following steps:
 +
 . Mirror the desired {product-title} image repository. For more information about how to mirror an image repository, see the "Mirroring the OpenShift Container Platform image repository" section.
```

Comment:
"Mirroring the OpenShift Container Platform image repository" should be changed to a link to existing doc. Also,
the linked instruction is about how to mirror the ocp image repository, so our description should also be updated to reflect that. 
Please see Angie's doc for more details.

[Mirroring the Openshift Container Platform image repository](https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images)

Reply:
> "Mirroring the OpenShift Container Platform image repository" should be changed to a link to existing doc

@yliu127 Unfortunately, we can't link there due to technical limitations, but I'll update/add a link in the `Additional resources` after the section.

Reply:
[Mirroring the Openshift Container Platform image repository](https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images) is from an existing ocp doc. And I saw links in procedures in this doc: https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images
Can we do the same?